x-pack/plugins/security/server/routes/api/public/roles/put.test.js

```diff
@@ -33,7 +41,7 @@ const putRoleTest = (
     const mockServer = createMockServer();
     const mockPreCheckLicense = jest
       .fn()
-      .mockImplementation(preCheckLicenseImpl);
+      .mockImplementation(preCheckLicenseImpl || defaultPreCheckLicenseImpl);
```

Comment:
Would you mind explaining why this is necessary and why the check for the `mockPreCheckLicense` not being called in the else below was removed?

Reply:
This is a good catch and something I meant to rollback after I had finished fixing this code up. I'm going to do another read pass through my test changes to make sure I didn't miss anything else.